backend/LexBoxApi/Services/EmailService.cs

```diff
@@ -166,8 +166,9 @@ private async Task SendInvitationEmail(
         var httpContext = httpContextAccessor.HttpContext;
         ArgumentNullException.ThrowIfNull(httpContext);
 
-        var queryString = QueryString.Create("email", emailAddress);
-        var returnTo = new UriBuilder { Path = "/acceptInvitation", Query = queryString.Value }.Uri.PathAndQuery;
+        var returnTo = _linkGenerator.GetUriByAction(httpContext,
+            nameof(LexBoxApi.Controllers.UserController.HandleInviteLink),
```

Comment:
This `nameof` is so that we can easily rename HandleInviteLink later if we need to. Can't do `nameof(UserController)` in the next line, though, but we're not going to rename the controllers so it's fine.